integration/client.go

```diff
@@ -22,3 +28,23 @@ func getK8sClient() (kubernetes.Interface, error) {
 
 	return cs, nil
 }
+
+type AWSClient struct {
+	EC2 *ec2.EC2
+}
+
+func getAWSClient() AWSClient {
```

Comment:
This is still confusing for me. Why is it called `getAWSClient` instead `newAWSClient`?
Why is private when struct it constructs is exported? I don't see AWSClient returned outside integration package.

Reply:
Yes, the `get` in the name comes from the k8s client, and no need to export the client type, fixed.